pom.xml

```diff
@@ -2373,6 +2374,17 @@
       </activation>
     </profile>
 
+    <profile>
+      <id>flink1.17</id>
+      <properties>
+        <orc.flink.version>1.5.6</orc.flink.version>
+      </properties>
+      <activation>
+        <property>
+          <name>flink1.17</name>
+        </property>
+      </activation>
+    </profile>
     <profile>
       <id>flink1.16</id>
       <properties>
```

Comment:
Need to fix these property items for 1.16 profile:

```xml
        <flink.version>${flink1.16.version}</flink.version>
        <hudi.flink.module>hudi-flink1.16.x</hudi.flink.module>
        <flink.bundle.version>1.16</flink.bundle.version>
```

Reply:
Thanks for reviewing the patch. Addressed this.